src/ua/hillel/FindWordPosition.java

```diff
@@ -0,0 +1,14 @@
+package ua.hillel;
+
+public class FindWordPosition {
+    public static void main(String[] args) {
+        String source;
+        String target;
+        boolean b;
+
+        source = "Apple";
+        target = "Java";
+        b = source.contains(target);
```

Comment:
this  logic is incorrect, it should return a position (index) of a target string in a source string. Does it make sense to you? For example, assume that source = "method", target = "et", an answer would = 1
Please use `indexOf()` method to solve this problem